hosts/binarycache/configuration.nix

```diff
@@ -54,13 +55,28 @@
   };
 
   services.nginx = {
+    package = pkgs.nginxStable.override {
+      modules = [pkgs.nginxModules.zstd];
+    };
+    recommendedZstdSettings = true;
```

Comment:
I think `recommendedZstdSettings` already adds the required module if the docs are to be believed: https://search.nixos.org/options?channel=23.11&show=services.nginx.recommendedZstdSettings&from=0&size=50&sort=relevance&type=packages&query=zstd

Reply:
Again, just adapted the settings from source, not sure why they did it explicitly.

Reply:
See https://github.com/NixOS/nixpkgs/blob/fa9f817df522ac294016af3d40ccff82f5fd3a63/nixos/modules/services/web-servers/nginx/default.nix#L1194, you don't have to set `package`.